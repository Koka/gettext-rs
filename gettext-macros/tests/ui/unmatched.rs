use gettext_macros::*;

fn main() {
    gettext!("Hello, {!");
    gettext!("Hello, }!");
}
