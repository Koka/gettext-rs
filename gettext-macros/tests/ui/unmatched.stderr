error: unmatched `{` in format string
       note: if you intended to print `{`, you can escape it using `{{`
 --> tests/ui/unmatched.rs:4:14
  |
4 |     gettext!("Hello, {!");
  |              ^^^^^^^^^^^

error: unmatched `}` in format string
       note: if you intended to print `}`, you can escape it using `}}`
 --> tests/ui/unmatched.rs:5:14
  |
5 |     gettext!("Hello, }!");
  |              ^^^^^^^^^^^
